ux.symfony.com/importmap.php

```diff
@@ -184,7 +185,28 @@
     'scheduler' => [
         'version' => '0.23.0',
     ],
-    '@symfony/ux-live-component' => [
-        'path' => './vendor/symfony/ux-live-component/assets/dist/live_controller.js',
+    'tippy.js' => [
+        'version' => '6.3.7',
+    ],
```

Comment:
This is ok. But the site is becoming a bit of a frakenstein's monster - e.g. with tooltips that could be used from Bootstrap or tippy.js. For the redesign, we should be choosy: use our own recipes to create a single Stimulus controller for each problem / a Twig component and stick with that everywhere

Reply:
I'm 100% with you on this. 

But we need to choose lib with that constraint (i had to install it directly because i needed to branch tooltips dynamically, et not like we do with the bootstrap ones)

That would sometime create debates, but globally i'm also convinced 👍